psiminer-core/src/main/kotlin/storage/tree/JsonTreeStorage.kt

```diff
@@ -68,7 +68,7 @@ class JsonTreeStorage(
         val nodeRepresentations = collectNodeRepresentation(labeledTree.root)
         ProjectManager.getInstance()
         val path = if (withPaths) labeledTree.root.getUserData(PATH_KEY) else null
-        val treeRepresentation = TreeRepresentation(labeledTree.label, path, nodeRepresentations)
+        val treeRepresentation = TreeRepresentation(labeledTree.label as String, path, nodeRepresentations)
```

Comment:
After changes in `LabelExtractor.kt` we don't need to change this line. Only add star projection to `labeledTree` in method arguments.